### PR TITLE
edit get message time

### PR DESCRIPTION
**Pull request by 19mdavenport**
Do you think this is a good idea to add? This would hopefully prevent students from timing out on the phase 6 tests and not needing to let tests run for 3.5 minutes.

### CODE REVIEW

File: src/main/java/edu/byu/cs/autograder/compile/CompileHelper.java [hunk 1]
```diff
@@ -92,4 +99,18 @@ private String getMavenError(String output) {
         }
         return builder.toString();
     }
+
+    private void replaceGetMessageTime() throws GradingException {
+        String regex = "return [0-9]*L;";
```
Comment (pawlh) [inline src/main/java/edu/byu/cs/autograder/compile/CompileHelper.java:104]: Shouldn't this be`[0-9]+L`?

File: src/main/java/edu/byu/cs/autograder/compile/CompileHelper.java [hunk 2]
```diff
@@ -92,4 +99,18 @@ private String getMavenError(String output) {
...
+        String replace = "return " + WEBSOCKET_MESSAGE_TIME + "L;";
+        Path path = new File(gradingContext.stageRepo(),
+                "/shared/src/test/java/passoffTests/TestFactory.java").toPath();
+        try {
+            String orig = Files.readString(path, StandardCharsets.UTF_8);
+            String replaced = orig.replaceAll(regex, replace);
+            Files.writeString(path, replaced);
+        } catch (Exception e) {
+            throw new GradingException("Failed to modify TestFactory", e);
```
Comment (pawlh) [inline src/main/java/edu/byu/cs/autograder/compile/CompileHelper.java:113]: Are students allowed to move this file? If not that makes things easier, but either way I think we should give a better error message, being more explicit about what is missing from where

**Review by pawlh — COMMENTED** [review]
I left a couple couple, but I think this is a good change

File: src/main/java/edu/byu/cs/autograder/compile/CompileHelper.java (hunk 2 above)
Comment (19mdavenport) [reply src/main/java/edu/byu/cs/autograder/compile/CompileHelper.java:113]: Hmm. I wouldn't think so, but I'm not sure. I guess there was a way easier way to change this too.

**Pull request closed without merging**